tests/FVTTests/testConfig.ts

```diff
@@ -0,0 +1,30 @@
+import { getDriver as getBrowserDriver, setApimlAuthTokenCookie  } from 'explorer-fvt-utilities';
+import { WebDriver } from 'selenium-webdriver';
+import { DEFAULT_SEARCH_FILTERS, loadPageWithFilterOptions } from './utilities';
+
+const {
+    ZOWE_USERNAME: USERNAME, ZOWE_PASSWORD: PASSWORD, SERVER_HOST_NAME, SERVER_HTTPS_PORT, TEST_BROWSER
+} = process.env;
+
+const BASE_URL = `https://${SERVER_HOST_NAME}:${SERVER_HTTPS_PORT}`;
+const BASE_URL_WITH_PATH = `${BASE_URL}/ui/v1/explorer-jes`;
+const FILTER_BASE_URL = `${BASE_URL_WITH_PATH}/#/`;
+const VIEWER_BASE_URL = `${BASE_URL_WITH_PATH}/#/viewer`;
+const loadUrlWithSearchFilters = loadPageWithFilterOptions(FILTER_BASE_URL, DEFAULT_SEARCH_FILTERS);
+const ZOSMF_JOB_NAME = 'IZUSVR1';
+
+export async function getDriver() {
+    let driver: WebDriver;
+    try {
+      driver = await getBrowserDriver(TEST_BROWSER) as WebDriver;
```

Comment:
Indentation here is different to the rest of the repo, should be 4 spaces or 4 character tab for consistent look